rhino.yml

```diff
@@ -8,59 +8,76 @@ services:
     dev-lidar-vlp32c:
-       container_name: dev-lidar-vlp32c
-       image: chalmersrevere/opendlv-device-lidar-vlp32c-multi:v0.0.2
-       restart: always
-       network_mode: "host"
-       command: "--vlp32c_ip=0.0.0.0 --vlp32c_port=2368 --cid=100"
-
-    # Accessing a video stream using OpenCV.
-#    dev-camera-opencv:
-#        container_name: dev-camera-opencv
-#        image: chalmersrevere/opendlv-device-camera-opencv-multi:v0.0.10
-#        restart: always
-#        ipc: "host"
-#        network_mode: "host"
-#        volumes:
-#        - /tmp:/tmp
-#        command: "--camera=http://root:axiscamera@10.40.40.29/axis-cgi/mjpg/video.cgi\\?channel=0\\&.mjpg --width=640 --height=480 --freq=20 --name.i420=video0.i420 --name.argb=video0.argb"
+        container_name: dev-lidar-vlp32c
+        image: chalmersrevere/opendlv-device-lidar-vlp32c-multi:v0.0.3
+        restart: always
+        network_mode: "host"
+        cap_add:
+        - SYS_NICE
+        entrypoint: ""
+        command: "nice -n -7 /usr/bin/opendlv-device-lidar-vlp32c --vlp32c_ip=0.0.0.0 --vlp32c_port=2368 --cid=100"
+
+    # Accessing a video stream using OpenCV (Axis 10.40.40.29)
+    dev-camera-opencv:
+        container_name: dev-camera-opencv
+        image: chalmersrevere/opendlv-device-camera-opencv-multi:v0.0.10
+        restart: always
+        ipc: "host"
+        network_mode: "host"
+        environment:
+        - CLUON_SHAREDMEMORY_POSIX=1
+        volumes:
+        - /tmp:/tmp
+        cap_add:
+        - SYS_NICE
+        entrypoint: ""
+        command: "nice -n -5 /usr/bin/opendlv-device-camera-opencv --camera=http://root:axiscamera@10.40.40.29/axis-cgi/mjpg/video.cgi\\?channel=0\\&.mjpg --width=640 --height=480 --freq=20 --name.i420=video0.i420 --name.argb=video0.argb"
 
     # Encoder for camera-opencv(x264)
     # x264(for research) has the better performance;
-#    video-x264-encoder-amd64:
-#        container_name: video-x264-encoder-amd64
-#        build:
-#            context: https://github.com/chalmers-revere/opendlv-video-x264-encoder.git
-#            dockerfile: Dockerfile.amd64
-#        restart: on-failure
-#        network_mode: "host"
-#        ipc: "host"
-#        volumes:
-#        - /tmp:/tmp
-#        command: "--cid=100 --name=video0.i420 --width=640 --height=480"
+    video-x264-encoder-amd64:
+        container_name: video-x264-encoder-amd64
+        build:
+            context: https://github.com/chalmers-revere/opendlv-video-x264-encoder.git
+            dockerfile: Dockerfile.amd64
+        restart: always
+        network_mode: "host"
+        ipc: "host"
+        environment:
+        - CLUON_SHAREDMEMORY_POSIX=1
+        volumes:
+        - /tmp:/tmp
+        cap_add:
+        - SYS_NICE
+        entrypoint: ""
+        command: "nice -n -10 /usr/bin/opendlv-video-x264-encoder --cid=100 --name=video0.i420 --width=640 --height=480"
 
     # Volvo Angle Sensor using lab jack
 #    dev-io-ue9-amd64:

```

Comment:
Did we test this?

Reply:
in a way, yes. It's still temporary but it works

Reply:
okay